include/fi_util.h

```diff
@@ -418,23 +433,26 @@ struct util_cmap {
 
 	struct ofi_key_idx key_idx;
 
+	uint8_t closing;
```

Comment:
A state that indicates a structure is about to be freed isn't useful unless there's some other reference counting occurring. I don't see this field being used.